lib/cached_resource/caching.rb

```diff
@@ -104,7 +104,7 @@ def is_any_collection?(*arguments)
       # Read a entry from the cache for the given key.
       def cache_read(key)
         object = cached_resource.cache.read(key).try do |json_cache|
-          json = ActiveSupport::JSON.decode(json_cache)
+          json = ActiveSupport::JSON.decode(MessagePack.unpack(json_cache))
```

Comment:
What do you think about making packing/unpacking another configurable option?

Reply:
Yeah thats what I was asking whether we should maintain it configurable or not. I can't think why anyone would not want this though.

If we make it configurable, we would have to `require` this dynamically based on the configuration option, which IMO is not an issue - just couldn't decide on whether this should be the norm or not.

I'll let you decide, configurable or norm? @mhgbrown

Reply:
Let's go configurable! Thanks for laying it out for me.

Reply:
Hey @jlurena, anything left to be done here?

Reply:
yeah the configuration part been a bit busy sorry

Reply:
No worries, thanks for the udpate. I see you made another commit. I'll check it out...

Reply:
Yep! Did as you asked.

Hey do you have discord or something? I'm hoping you can help me setup a `pre` version of `cached_resource` that makes use of this version of ActiveResource:

https://github.com/rails/activeresource/pull/409

Reply:
Hey yeah, my handle is mhgbrown